app/CMakeLists.txt

```diff
@@ -335,9 +335,22 @@ endif()
 #-----------------------------------------------------------------------------#
 # DEMO: Geant4 integration
 #-----------------------------------------------------------------------------#
+function(celeritas_reflex_generate_dictionary)
+  # Inform ROOT scripts of include directories since it can't pull them through
+  # targets
+  include_directories(
```

Comment:
Note: This is another difference, `reflex_generate_dictionary` is missing (for now?) the code that exist in `root_generate_dictionary` to propagate this information.